reference/mongodb/bson/iterator/valid.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- $Revision$ -->
+<refentry xml:id="mongodb-bson-iterator.valid" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+ <refnamediv>
+  <refname>MongoDB\BSON\Iterator::valid</refname>
+  <refpurpose>Checks if current position is valid</refpurpose>
+ </refnamediv>
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis>
+   <modifier>public</modifier> <type>bool</type><methodname>MongoDB\BSON\Iterator::valid</methodname>
+   <void/>
+  </methodsynopsis>
+ </refsect1>
+
+ <refsect1 role="parameters">
+  &reftitle.parameters;
+  &no.function.parameters;
+ </refsect1>
+
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
+   Returns &true; if the iterator is valid, &false; otherwise.
```

Comment:
Consider replacing "iterator" with one of the following:

 * "current position"
 * "current iterator position"
 * "iterator's current position"

There is prior art for the first two bullets in other `Iterator::valid()` implementations within the PHP docs.

Reply:
Replaced with "current iterator position".